os/board/rtl8721csm/src/component/os/tizenrt/rtk_ble_utils.h

```diff
@@ -54,9 +54,12 @@ extern trble_result_e rtw_ble_server_delete_bonded_device(uint8_t bd_addr[TRBLE_
 extern trble_result_e rtw_ble_server_delete_bonded_device_all(void);
 extern trble_result_e rtw_ble_server_set_adv_interval(uint16_t interval); // interval range (0x0020 ~ 0x4000)
 
+extern trble_result_e rtw_ble_client_scan_whiltelist_add(trble_addr *addr);
+extern trble_result_e rtw_ble_client_scan_whiltelist_delete(trble_addr *addr);
+extern trble_result_e rtw_ble_client_scan_whiltelist_clear_all(void);
```

Comment:
I think these are typos.
whiltelist -> whitelist

Reply:
Yes, it is a typo, thank for pointing out.
Corrected in latest commit.